### PR TITLE
rafs: reduce log level to avoid annoying messages on console

### PR DESCRIPTION
**Pull request by jiangliu**
## Details
Reduce log level to avoid annoying messages on console such as: root@liujiang-ecs-dev:~/ws/nydus-snapshotter.git/cmd/containerd-nydus-grpc# nerdctl run --rm -ti --snapshotter nydus busybox
docker.io/library/busybox:latest:                                                 resolved       |++++++++++++++++++++++++++++++++++++++|
index-sha256:6e494387c901caf429c1bf77bd92fb82b33a68c0e19f6d1aa6a3ac8d27a7049d:    done           |++++++++++++++++++++++++++++++++++++++|
manifest-sha256:1b0a26bd07a3d17473d8d8468bea84015e27f87124b283b91d781bce13f61370: done           |++++++++++++++++++++++++++++++++++++++|
config-sha256:b539af69bc01c6c1c1eae5474a94b0abaab36b93c165c0cf30b7a0ab294135a3:   done           |++++++++++++++++++++++++++++++++++++++|
elapsed: 11.3s                                                                    total:  4.2 Ki (378.0 B/s)
[2023-06-27 17:51:14.443766 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: rafs/src/metadata/md_v6.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ impl RafsSuper {
 
         self.meta.flags = RafsSuperFlags::from_bits(ext_sb.flags())
             .ok_or_else(|| einval!(format!("invalid RAFS flags 0x{:x}", ext_sb.flags())))?;
-        info!("RAFS features: {}", self.meta.flags);
+        debug!("RAFS features: {}", self.meta.flags);
```
Comment (imeoer) [inline rafs/src/metadata/md_v6.rs:51]: It seems useful when we use `nydus-image check`.

Comment (adamqqqplay) [reply rafs/src/metadata/md_v6.rs:51]: > It seems useful when we use `nydus-image check`.

I think so.

Comment (jiangliu) [reply rafs/src/metadata/md_v6.rs:51]: It generates annoying message when running containers:
root@liujiang-ecs-dev:~/ws/nydus-snapshotter.git/cmd/containerd-nydus-grpc# nerdctl run -ti --rm --snapshotter nydus nginx
docker.io/library/nginx:latest:                                                   resolved       |++++++++++++++++++++++++++++++++++++++| 
index-sha256:593dac25b7733ffb7afe1a72649a43e574778bf025ad60514ef40f6b5d606247:    done           |++++++++++++++++++++++++++++++++++++++| 
manifest-sha256:d2b2f2980e9ccc570e5726b56b54580f23a018b7b7314c9eaff7e5e479c78657: done           |++++++++++++++++++++++++++++++++++++++| 
config-sha256:eb4a57159180767450cb8426e6367f11b999653d8f185b5e3b78a9ca30c2c31d:   done           |++++++++++++++++++++++++++++++++++++++| 
elapsed: 42.3s                                                                    total:  11.5 K (278.0 B/s)                                       
[2023-06-29 21:43:19.647693 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.828587 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.920714 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921195 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921417 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921645 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
[2023-06-29 21:43:19.921847 +08:00] INFO RAFS features: COMPRESSION_NONE | HASH_SHA256 | EXPLICIT_UID_GID | TARTFS_MODE
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh

**Comment by jiangliu** [comment]
I will solve the issue in another way, so close this one.

**Pull request closed without merging**